Sources/Shared/Change+Map.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Change+Map.swift
+//  DeepDiff
+//
+//  Created by Javier Osorio Goenaga on 2/26/19.
+//  Copyright © 2019 Khoa Pham. All rights reserved.
+//
+
+import Foundation
+
+public extension Change {
+    public func map<U>(_ transform: (T) throws -> U) rethrows -> Change<U> {
```

Comment:
@jrogccom can we go with 2 spaces ?

Reply:
@onmyway133  Absolutely.  I'll do it manually, but is there a linter you use?

Reply:
I haven't configured SwiftLint for this